### DISCUSSION TITLE
Issue with Accessing PocketHost Admin Page After Login

### DISCUSSION THREAD

**Discussion by Hanna-Jeanne** [pockethost/pockethost#413] [Q&A]
Hello,

I’m trying to use PocketHost for the first time to manage data. However, after logging in and accessing the admin page, I encounter the following error:

"Please contact PocketHost support."

I’ve already posted a support inquiry regarding this issue. Could you please guide me on how to resolve this?

Thank you.

**Comment by MinQyu** [comment]
I'm having the same problem

**Comment by benallfree** [comment]
Please email ben@pockethost.io to resolve.
